subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/ecb.c

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2016-2017 Nordic Semiconductor ASA
  * Copyright (c) 2016 Vinayak Kariappa Chettimada
- * Copyright 2019 NXP
+ * Copyright 2019-2020 NXP
```

Comment:
Should be some spaces in here (could update line 2 as well)

Reply:
Don't need the spaces, and don't modify someone else's copyright.